### PR TITLE
Add mainnet chains and token config unit-tests

### PR DESCRIPTION
**Pull request by claudijd**
These unit-tests are mainly just to ensure a mainnet config update doesn't result in a regression of config expectations.

- - -
This change is [Reviewable](https://reviewable.io/reviews/wormhole-foundation/wormhole/1461)

### CODE REVIEW

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:34]: To make this a bit more future-proof we could use `common.KnownTokenbridgeEmitters`.  That way this test will fail if a new tokenbridge emitter is added to the list without a corresponding governor config.

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:26]: style nit: we can do `50_000_001` to make this constant a bit more readable.

File: node/pkg/governor/mainnet_tokens_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_tokens_test.go:47]: Same comment as above about using `common.KnownTokenbridgeEmitters`.

File: node/pkg/governor/mainnet_tokens_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_tokens_test.go:78]: style nit: the idiomatic way to check for duplicates in go is to use a set:

```suggestion
	addrs := make(map[string]bool)
	for _, e := range tokenConfigEntries {
		assert.False(t, addrs[e.addr])
		addrs[e.addr] = true
	}
```

Here the assertion relies on the fact that [looking up a non-existent key in the map will return the zero value](https://go.dev/doc/effective_go#maps) (false for bools)

File: node/pkg/governor/mainnet_tokens_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_tokens_test.go:88]: nit:

```suggestion
		d := tokenConfigEntry.decimals
		assert.Condition(t, func() bool { return d == 6 || d == 8 })
```

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:49]: The assert package has both `Subset` and `ElementsMatch` methods:

```suggestion
	entries := make([]vaa.ChainID, 0, len(chainConfigEntries))
	for _, e := range chainConfigEntries {
		entries = append(entries, e.emitterChainID)
	}
	
	emitters := make([]vaa.ChainID, 0, len(common.KnownTokenbridgeEmitters))
	for e := range common.KnownTokenbridgeEmitters {
		emitters = append(emitters, e)
	}
	
	assert.ElementsMatch(t, entries, emitters)
```

**Comment by claudijd** [comment]
@jynnantonix awesome review, thank you! Fixes are up!

File: node/pkg/governor/mainnet_tokens_test.go
Comment (SEJeff) [inline node/pkg/governor/mainnet_tokens_test.go:49]:
```suggestion
	/* Assume that all chains with a token bridge will have governed tokens */
```

Pythnet (#1459) does not have a token / nft bridge

File: node/pkg/governor/mainnet_tokens_test.go
Comment (SEJeff) [inline node/pkg/governor/mainnet_tokens_test.go:89]:
```suggestion
	/* Assume that all governed token entry symbol strings will be greater than zero */
```

File: node/pkg/governor/mainnet_tokens_test.go
Comment (SEJeff) [inline node/pkg/governor/mainnet_tokens_test.go:98]:
```suggestion
	/* Assume that all governed token entry coingecko id strings will be greater than zero */
```

**Review by SEJeff — APPROVED** [review]

File: node/pkg/governor/mainnet_chains_test.go
Comment (claudijd) [reply node/pkg/governor/mainnet_chains_test.go:17]: fine with me, will do

File: node/pkg/governor/mainnet_chains_test.go
Comment (claudijd) [reply node/pkg/governor/mainnet_chains_test.go:38]: can do

**Review by SEJeff — APPROVED** [review]

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:24]: "value movement"?

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:28]: `min_daily_limit` is 0 so doesn't this fail every time?  Is this supposed to be for `max_daily_limit`?

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:23]: "These" -> "This"

File: node/pkg/governor/mainnet_chains_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_chains_test.go:30]: "These" -> "This", "them" -> "it"

File: node/pkg/governor/mainnet_tokens_test.go
Comment (jynnantonix) [inline node/pkg/governor/mainnet_tokens_test.go:51]: We can do the same thing here that we did for `TestChainListChainPresent`:

```
	entries := make([]vaa.ChainID, 0, len(tokenConfigEntries))
	for _, e := range tokenConfigEntries {
		entries = append(entries, vaa.ChainID(e.chain))
	}

	emitters := make([]vaa.ChainID, 0, len(common.KnownTokenbridgeEmitters))
	for e := range common.KnownTokenbridgeEmitters {
		emitters = append(emitters, e)
	}

	assert.ElementsMatch(t, entries, emitters)
```

I would make this a suggestion but github in its infinite wisdom doesn't allow comments outside of the diff range 🤦.

**Review by jynnantonix — APPROVED** [review]
lgtm with nits

File: node/pkg/governor/mainnet_tokens_test.go
Comment (claudijd) [reply node/pkg/governor/mainnet_tokens_test.go:51]: `tokenConfigEntries` will have duplicate chainIDs in it, so if we compare `entries` (has duplicates) to `emitters` (doesn't have duplicates) it will not match.  This is why I'm keeping the existing strategy on the tokens vs. the chains code where I did make it so.

**Review by SEJeff — APPROVED** [review]

**Comment by claudijd** [comment]
now blocked by https://github.com/wormhole-foundation/wormhole/pull/1452 (the schema expectation has changed, might want to add another set of checks for big transfers)

File: node/pkg/governor/mainnet_tokens_test.go
Comment (jynnantonix) [reply node/pkg/governor/mainnet_tokens_test.go]: In that case we could use a `map[vaa.ChainID]bool` to deduplicate before comparing but I'm also fine with leaving this as is.

**Review by jynnantonix — APPROVED** [review]

File: node/pkg/governor/mainnet_tokens_test.go
Comment (claudijd) [reply node/pkg/governor/mainnet_tokens_test.go]: Ok, that makes sense, I'll make that change once we settle on the auto-merge PR and I adapt this to include big tx unti-text expectations.

**Comment by jynnantonix** [comment]
Looks like the checks were stale because a couple of these tests are broken on dev.v2. See #1485 for the fix.

**Pull request merged**
